<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FieldDlg</class>
 <widget class="QDialog" name="FieldDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>573</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>266</width>
    <height>390</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Field</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="nameLabel">
             <property name="text">
              <string>&amp;Name:</string>
             </property>
             <property name="buddy">
              <cstring>nameLineEdit</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nameLineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QPushButton" name="linkDSPushButton">
             <property name="text">
              <string>&amp;Link DataSource</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="applyPushButton">
           <property name="text">
            <string>A&amp;pply</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="resetPushButton">
           <property name="text">
            <string>Re&amp;set</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="unitsLabel">
         <property name="text">
          <string>&amp;Units:</string>
         </property>
         <property name="buddy">
          <cstring>unitsLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="unitsLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="valueLabel">
         <property name="text">
          <string>&amp;Value:</string>
         </property>
         <property name="buddy">
          <cstring>valueLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="valueLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item row="0" column="0">
        <widget class="QFrame" name="typeFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="typeLabel">
              <property name="text">
               <string>&amp;Type:</string>
              </property>
              <property name="buddy">
               <cstring>typeComboBox</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="typeComboBox">
              <item>
               <property name="text">
                <string>None</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>NX_CHAR</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>NX_BOOLEAN</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>NX_INT</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>NX_FLOAT</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>NX_UINT</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>NX_INT8</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>NX_INT16</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>NX_INT32</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>NX_INT64</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>NX_FLOAT32</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>NX_FLOAT64</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>NX_UINT8</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>NX_UINT16</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>NX_UINT32</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>NX_UINT64</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>other ...</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
         <zorder></zorder>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QFrame" name="otherFrame">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QLineEdit" name="typeLineEdit"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="dimPushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Di&amp;mensions:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="dimLabel">
         <property name="text">
          <string> []              </string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="attributesLabel">
         <property name="text">
          <string>&amp;Attributes:</string>
         </property>
         <property name="buddy">
          <cstring>attributeTableWidget</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addPushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>A&amp;dd</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removePushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Remove</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="attributeTableWidget"/>
     </item>
     <item>
      <widget class="QLabel" name="docLabel">
       <property name="text">
        <string>&amp;Doc:</string>
       </property>
       <property name="buddy">
        <cstring>docTextEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="docTextEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>linkDSPushButton</tabstop>
  <tabstop>unitsLineEdit</tabstop>
  <tabstop>valueLineEdit</tabstop>
  <tabstop>typeComboBox</tabstop>
  <tabstop>typeLineEdit</tabstop>
  <tabstop>dimPushButton</tabstop>
  <tabstop>addPushButton</tabstop>
  <tabstop>removePushButton</tabstop>
  <tabstop>attributeTableWidget</tabstop>
  <tabstop>docTextEdit</tabstop>
  <tabstop>applyPushButton</tabstop>
  <tabstop>resetPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
